modules/custom/openy_pef_gxp_sync/README.md

```diff
@@ -48,3 +48,7 @@ The syncer consists of the next steps:
 `@openy_pef_gxp_sync.fetcher_debugger` to emulate API response.
 
 2. Use `DEBUG_MODE` constants inside classes to debug specific service.
+
+### Known issues in sync.
```

Comment:
It's really a feature. `General` means that class is not categorised.
Probably we have to add todo.
I'd add a feature which will optionally add/remove sync for `general` category.